chains/sql_database_test.go

```diff
@@ -45,3 +46,106 @@ func TestSQLDatabaseChain_Call(t *testing.T) {
 
 	t.Log(ret)
 }
+
+func TestExtractSQLQuery(t *testing.T) {
+	t.Parallel()
+
+	// mock single line sql query.
```

Comment:
instead of defining these cases as named params please inline them with the set of cases defined (similar to my comment on https://github.com/tmc/langchaingo/pull/614 )